packages/website/docs/TextInput.md

```diff
@@ -1,20 +1,43 @@
-## Guidelines
+[Description]: # 
+[Text inputs allow to enter and edit short texts, such as names, emails or passwords.]: # 
+>
+> ### Best Practices :bulb: 
```

Comment:
I think we'll need to add svg or icons in the repo if we want to use them in the guidelines :thinking: 
:bulb: is not rendering in the demo.

Reply:
Don't bother, I will remove them, it was an hopeful test but not required for now.

Reply:
real emojis should work